README.md

```diff
@@ -57,6 +57,21 @@ where
  * `cleanup_interval` (default: 1 hour) is the number of seconds between database cleanups 
  * `metrics_lifetime` (default: 24 hours) is the number of seconds that must pass from creation before each metric can be deleted during cleanup
 
+## Metrics visualisation with Grafana
+
+Metrics stored in the database using `membrane_rtc_engine_timescaledb` can be simply visualized using Grafana.
+To start the dashboard with RTC Engine metrics, pass volume with the content of `priv/grafana/provisioning` to the Grafana docker container.
+You can see, how it is done in this [docker-compose.yml](https://github.com/membraneframework/membrane_videoroom/blob/metrics-visualisation/docker-compose.yml)
```

Comment:
```suggestion
You can see, how it is done with the use of the `grafana-provisioning` volume in this [docker-compose.yml](https://github.com/membraneframework/membrane_videoroom/blob/metrics-visualisation/docker-compose.yml)
```

Reply:
The "suggested change" proposed in the comment above is one of the approaches I suggest using so that to help the user understand what is going on. The other approach might be to move the link to the exemplary docker-compose file to the bottom of this section, and emphasize that this configuration sets both the volume and the environmental variables.